src/tools/miri/tests/pass/binops.rs

```diff
@@ -2,23 +2,23 @@
 
 fn test_nil() {
     assert_eq!((), ());
-    assert!((!(() != ())));
-    assert!((!(() < ())));
-    assert!((() <= ()));
-    assert!((!(() > ())));
-    assert!((() >= ()));
+    assert!(!(() != ()));
```

Comment:
All of these are because now the unnecessary parentheses lint (correctly) triggers on `assert!((expr))`.